src/commands/config/prompts.js

```diff
@@ -11,11 +13,12 @@ export const EMOJI_COMMIT_FORMATS = {
   EMOJI: 'emoji'
 }
 
-export default [
+export default () => [
```

Comment:
Why do we need to convert this into a function?

Reply:
This object is not longer immutable since it depend on `configurationVault` values. Changing it to a function ensures that it always use the current config.

eg:
```
configurationVault.setAutoAdd(true)
// configurationPrompts()[0].default is true
configurationVault.setAutoAdd(false)
// configurationPrompts()[0].default is false
```